_extras/snippets_library/UCL_sge/14/matlab_example.snip

```diff
@@ -0,0 +1,138 @@
+Let's take a closer look at the `matlab` module.
```

Comment:
to keep it more general with the rest of locations, rename this file as in the original gcc example.

Reply:
I don't understand what you mean, sorry.

Reply:
Save the content of this file under `_extras/snippets_library/UCL_sge/14/gcc_example.snip` so there's no need to change the markdown file.

Reply:
Done, in commit 6cdc44a.